src/libs/ReportUtils.ts

```diff
@@ -1667,14 +1667,6 @@ function canAddOrDeleteTransactions(moneyRequestReport: OnyxEntry<Report>): bool
         return false;
     }
 
-    if (isIOUReport(moneyRequestReport) && currentUserAccountID !== moneyRequestReport?.managerID && currentUserAccountID !== moneyRequestReport?.ownerAccountID) {
```

Comment:
This is bad code form a PR I'm reverting here https://github.com/Expensify/App/pull/47595

Reply:
@youssef-lr Since we just merged the other PR, can you update this branch with main to make sure everything looks right?